src/styles/main.scss

```diff
@@ -1,3 +1,212 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+  width: 100%;
+}
+
+.main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  align-self: center;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  color: #060b35;
+  position: relative;
+  display: block;
+  padding: 23px 0;
+  font-weight: 500;
+  line-height: 14px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__link[data-qa="nav-hover"]:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+
+  &::after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    height: 4px;
+    width: 100%;
+    background: #00acdc;
+    border-radius: 12px;
+  }
+}
+
+p,
+h3 {
+  margin: 0;
+}
+
+.card-container {
```

Comment:
This style has `card.scss` file, so remove it here. Fix all cases

Reply:
I no longer use the main.scss file; instead, I use all.scss. 
The main file has been renamed to all.scss.